hadoop-ozone/integration-test/src/test/resources/log4j.properties

```diff
@@ -19,3 +19,6 @@ log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} (%F:%
 
 log4j.logger.org.apache.hadoop.security.ShellBasedUnixGroupsMapping=ERROR
 log4j.logger.org.apache.hadoop.util.NativeCodeLoader=ERROR
+
+log4j.logger.org.apache.ratis.grpc.server.GrpcLogAppender=DEBUG
+log4j.logger.org.apache.ratis.server.impl.RaftServerImpl=DEBUG
```

Comment:
Hi @xiaoyuyao,

Were these Ratis log levels intentionally set to DEBUG in the final commit, or is it a leftover that should have been removed?